### ISSUE TITLE
Do users want to be able to store tissue type in spaceTx-format? 

### ISSUE THREAD

**Issue by ambrosejcarr** [spacetx/starfish#715]
It came up in conversation with @zperova and @hewgreen that many of our experiments may encompass borders between defined tissue regions. Larger atlases may want to reference only the data that corresponds to one tissue or the other, and users may want to be able to reference slices that correspond to only one part of the sample. 

Do users want to store data like this in spaceTx format? Would this be helpful? 

@joshmoore do you have experience with users requesting this granularity of annotation? 
@berl would you use these fields if they were provided, for example, to separate out the different layers of cortex?

**Comment by berl** [comment]
tl;dr: I don't think this belongs in the spacetx format.

are we talking about layers?  If so, each group's contribution should cover all cortical layers.  We were planning to annotate this at the Cell x Gene level (e.g. each cell gets assigned to a layer and also gets a layer depth coordinate), based on the overview images that contributors provide. I haven't thought about annotating the raw images or dividing up the raw image space into different anatomical regions.   

So for layers, I would think that detailed annotation all the way through the data would be difficult and probably not necessary.  Layer annotation for the cells is required, but seems to fit better as a tertiary analysis activity.  

For cortical region, We (spacetx) are planning to only analyze MTG in human and V1 in mouse. If someone submits an entire coronal slice of mouse that would be impressive, but we would not try to annotate all the structures and analyze them- we'd just pull out V1 for comparison.

Now that I write this (very longwinded) response, it's clear that a long-term solution in this area will require attention both from metadata folks for laying out the hierarchy and from @joshmoore  / OMERO for tools to actually annotate stuff.

**Comment by hewgreen** [comment]
> each cell gets assigned to a layer and also gets a layer depth coordinate
This is cool. I need to learn more about this.

> dividing up the raw image space into different anatomical regions
We divide up a bit with support from multiple hybridization_FOV's right? Though I see how different regions can be in one FOV. We'd just list them. If we are thinking about an experiment with 1 FOV in 1 experiment.json then the rest of my post is mute.

I just want to know what granularity of a priori region annotation is useful to downstream consumers. Based on the above you seem to be saying that each layer is useful. It should go somewhere, if not in starfish metadata I'm unclear where it lives prior to analysis?

At the moment, any metadata at a finer grain than experiment.json must be captured by the starfish format. If any of this finer grain metadata is useful to search across datasets it needs to be extracted for indexing as the HCA is proposing for gene targets. But there we link a list of targets to one experiment.json. Here we are asking if a user would want to extract tissue region labels and link them to FOV rather than to a whole experiment? If so the implication is that tertiary metadata needs to point to sub manifests not just the experiment.json which has consequences on starfish's definition of experiment.json.

Would 1. be ok for a cross dataset data consumer or is 2. required?
1. _a bucket of files defined by experiment.json that has a list of their query regions that were labeled added a priori_
2. _a FOV is region labelled x, then maybe a separate FOV region labelled x-y_

**Comment by berl** [comment]
@hewgreen A couple ideas:
1. the anatomical annotation is a list of regions sampled in the experiment, which would live in the HCA metadata and perhaps be indexed
2. based on the overview image (requested as part of HCA metadata and SpaceTx project) and annotations on it (possibly with josh's help) the polygons associated with layers and/or regions are mapped to the coordinates of the analyzed data.

In our hands, the annotation is often best done on the largest image we have and then translated down into the coordinates of the analyzed data.  We typically do this and also add a single line indicating the cortical column axis (roughly perpendicular to layers). x,y positions can be projected against this axis to generate a cortical depth coordinate.  I'll post a picture on the spacetx slack

**Comment by joshmoore** [comment]
> @joshmoore do you have experience with users requesting this granularity of annotation?

As a separate image, or as a portion of an image. If the latter certainly, annotations on a region of interest. The former sounds like a derived image.

To @berl's https://github.com/spacetx/starfish/issues/715#issuecomment-430301306, something that isn't overly exposed as of yet but could be of interest would be hierarchies of those regions (all annotatable).

> At the moment, any metadata at a finer grain than experiment.json must be captured by the starfish format (https://github.com/spacetx/starfish/issues/715#issuecomment-430318622)

This matches what I'm thinking above. Certainly based on the "Image Submission Structure" drawing, I'd minimally expect to be able to capture those types of annotations in the OME-XML.

> based on ... and annotations on it (possibly with josh's help) the polygons associated with layers and/or regions are mapped to the coordinates of the analyzed data. (https://github.com/spacetx/starfish/issues/715#issuecomment-430343922)

In general, for anything which OME considers an Image which maps to a SpaceTx FOV, tooling exists for defining regions & annotations.

**Comment by berl** [comment]
> In general, for anything which OME considers an Image which maps to a SpaceTx FOV, tooling exists for defining regions & annotations.
@joshmoore  does this include annotations that span FOVs?  each layer annotation in this example
https://slack-files.com/T87CZHS5N-FDEQNRXS4-854830927e
goes across several FOVs

**Comment by joshmoore** [comment]
(In the OME-sense) A single ROI doesn't span multiple Images, no.
